src/neo/SmartContract/Native/Oracle/OracleContract.cs

```diff
@@ -0,0 +1,224 @@
+using Neo.Cryptography.ECC;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.SmartContract.Manifest;
+using Neo.SmartContract.Native.Tokens;
+using Neo.VM.Types;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Array = Neo.VM.Types.Array;
+
+namespace Neo.SmartContract.Native
+{
+    public class OracleContract : NativeContract
```

Comment:
`OracleContractAsync`, if it is just for Committee and not open for other to register maybe it should be `OracleContractAsyncFixed`, something like that, until we allow other to participate.